l10n_de_tax_statement/README.rst

```diff
@@ -0,0 +1,73 @@
+
+#. Verify that you have enough permits. You need to belong at least to the Accountant group.
+#. Go to the menu: Invoicing -> Reports > Taxes Balance > DE Tax Statement
+#. Create a statement, provide a name and specify start date and end date
+#. Press the Update button to (re-) calculate the report: the report lines will be displayed in the tab Statement
+#. Eventually you have to manually enter the Tax amounts of lines in Edit mode, click on the amount of the line to be able to change the values (f.e. to remove decimal values) .
+#. Press the Post button to set the status of the statement to Posted; the statements set to this state cannot be modified anymore
+#. If you need to recalculate or modify or delete a statement already set to Posted status you need first to set it back to Draft status: press the button Reset to Draft
+#. If you need to print the report in PDF, open a statement form and click: Print -> DE Tax Statement
+#. On the second tab of the statement you have the possibility to add undeclared invoices from the past to the VAT statement. By pressing the "Update" button you will add the in the statement's report.
+
+
+
+Known issues / Roadmap
+======================
+
+* Exporting formats not yet available
+* If you have used the V10 version of that module, it might need some adoptions as the tax lines of the statement report was revised for the V11 module.
+* If the usual german tax rates would become more expensive or even cheaper as 19% and 7% the module would need some refactoring, in order to reflect those tax adoptions in order to calculate proper values. Eventually we can improve the module by fetching the tax values from some taxes instead of calculting these values in order to prepare the tax fields of this report. However to do that we need to assign for each tax in the german account plan (modules l10n_de_skr03 and l10n_de_skr04) a tax tag for the base amount as well also the tax tag for the tax value, which is currently not the case. Currently we have assigned to many taxes in the mentioned localisation modules just one tax tag. Nevertheless the tax calculation in this module should not differ from the values, if we would fetch them by tax tags for the tax amount.
+* Maybe we could add more german specific tax cases inside the "test" directory.
+
+
+
+Credits
+=======
+
+Contributors
+------------
+* Thorsten Vocks <thorsten.vocks@openbig.org>
+* Andrea Stirpe <a.stirpe@onestein.nl>
+* Antonio Esposito <a.esposito@onestein.nl>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is not yet maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit https://odoo-community.org or the corresponding github repository. 
```

Comment:
Could you please add the readme folder, following the template https://github.com/OCA/maintainer-tools/tree/master/template/module/readme, ad copy the content of this `README.rst` into fragment files in the readme folder?

See https://github.com/OCA/l10n-netherlands/tree/11.0/l10n_nl_tax_statement/readme

Reply:
I have applied your proposed change. Thanks a lot for your intervention!